README.md

```diff
@@ -44,6 +44,7 @@
 | [Hy](#hy) | [Joel Martin](https://github.com/kanaka)  |
 | [Io](#io) | [Dov Murik](https://github.com/dubek) |
 | [Java](#java-17) | [Joel Martin](https://github.com/kanaka)  |
+| [Jq](#jq) | [Ali MohammadPur](https://github.com/alimpfard) |
```

Comment:
I think Jq should be just before Julia according to alphabetic order.
Also please update the total numbers at the top of the table.